storio/src/androidTest/java/com/pushtorefresh/storio/db/integration_test/impl/DeleteTest.java

```diff
@@ -45,4 +41,36 @@
         assertEquals(0, cursorAfterDelete.getCount());
         cursorAfterDelete.close();
     }
+
+    @Test public void deleteCollection() {
+        final List<User> allUsers = putUsers(10);
+
+        final List<User> usersToDelete = new ArrayList<>();
+        for (int i = 0; i < allUsers.size(); i += 2) {  // I will delete every second
+            usersToDelete.add(allUsers.get(i));
+        }
+
+        final DeleteCollectionOfObjectsResult<User> deleteResult = storIODb
+                .delete()
+                .objects(usersToDelete)
+                .withMapFunc(User.MAP_TO_DELETE_QUERY)
+                .prepare()
+                .executeAsBlocking();
+
+        final List<User> existUsers = storIODb
+                .get()
+                .listOfObjects(User.class)
+                .withMapFunc(User.MAP_FROM_CURSOR)
+                .withQuery(new Query.Builder()
+                        .table(User.TABLE)
+                        .build())
+                .prepare()
+                .executeAsBlocking();
+
+        for (User user : allUsers) {
+            final boolean deleted = usersToDelete.contains(user);
+            assertEquals(deleteResult.results().containsKey(user), deleted);
```

Comment:
Actually, it's a problem of `DeleteResult`, may be we should add a method for quick check `wasDeleted(object)` and `wasNotDeleted(object)`? It will be great for users and for writing tests! It can be part of PR :)